app/styles/_icons.scss

```diff
@@ -252,10 +252,15 @@ i.fa-circle-o-notch.fa-spin {
 }
 
 .icon--search {
-  width: 21px;
-  height: 21px;
+  width: 20px;
+  height: 23px;  
 }
 
+.searchbox--slim .icon--search {
+  width: 14px;
+  height: 15px;
+} 
+
```

Comment:
This is a good opportunity to refactor the search box css to use functional-semantic class names (`.site-nav-searchbox`) with design-semantic mixins (`@mixin gray-searchbox`), in the same manner that we've started to refactor the buttons file. let's talk a bit about it.